python/cuml/tsa/arima.pyx

```diff
@@ -127,11 +115,11 @@ class ARIMA(Base):
              + np.tile(pattern, (25, 1)))
 
         # Fit a seasonal ARIMA model
-        model = arima.ARIMA(y, (0,1,1), (0,1,1,4), fit_intercept=False)
+        model = ARIMA(y, (0,1,1), (0,1,1,4), fit_intercept=False)
         model.fit()
 
         # Forecast
-        fc = model.forecast(10).copy_to_host()
+        fc = model.forecast(10).to_output('numpy')
```

Comment:
`CumlArray` is not a method meant for user consumption, so this method shouldn't be in the docstring. See the docstring: https://github.com/rapidsai/cuml/blob/a9846960b44463114b7204903899c3683306cd55/python/cuml/common/base.pyx#L61 
the way that the formatting of results is handled is by using the `output_type` parameter logistic as described in the base docstring, so this needs to be added overall